s2spy/_RGDR/rgdr.py

```diff
@@ -0,0 +1,35 @@
+"""Response Guided Dimensionality Reduction."""
```

Comment:
Actually it is a bit tricky. I started with `RGDR/_rgdr.py`, but then it looks a bit weird that while `_rgdr` is not supposed to be accessed by the user, `map_analysis.py` and `map_regions` are visible to the user. As both of them are the utils for RGDR, it makes sense to me that they should be `_map_analysis.py` and `_map_regions`. Since everything is hidden, it seems more logical to have the entire folder invisible, and therefore the `_RGDR`.

I guess we still need to discuss it.

Btw, I will rename the folder as rgdr.